### ISSUE TITLE
Request: Regular ES5 version without Node

### ISSUE THREAD

**Issue by gordon-matt** [SparkPost/heml#11]
Could you guys build a regular ES5 version (without requiring Node) and have it under releases and/or available on Bower?

**Comment by avigoldman** [comment]
Yeah, that definitely is something we are looking into. I'd love to have this bundled for the browser. There are a couple of dependencies that don't play nicely with bundling (specifically in [`@heml/styles`])https://github.com/SparkPost/heml/tree/master/packages/heml-styles) but if we can iron those out we should be in good shape.

**Comment by gordon-matt** [comment]
@avigoldman Any idea when this can be done? I can't quite figure out how to use babel to bundle this myself. If you could bundle up something for me to get started with now (before this is officially supported in Bower or releases) so that I can get started on developing my app, I would be very grateful. If you're too busy, no worries but then any estimate on when it might be ready would be helpful. Thanks in advance.

**Comment by avigoldman** [comment]
One of the dependencies is not bundling nicely 😞 I'll talk to some of our babel pros at SparkPost and see if they know of any good solutions. The best option for the moment is to offload the HEML work onto Heroku, Lambda, or an Azure Function. I'll post updates to this as they come in.

**Comment by aydrian** [comment]
I know some people at @grovolearning that used a CI solution like Travis to compile their templates and add/update them as a stored template in SparkPost on Github commit. That could be an easy solution if you have a template store.

**Comment by adamlc** [comment]
Any updates on this? We'd like to integrate this into our web app if possible 👍
